core/src/main/scripts/importer/validateData.py

```diff
@@ -1192,6 +1181,7 @@ class ClinicalValidator(Validator):
     REQUIRE_COLUMN_ORDER = False
     PROP_IS_PATIENT_ATTRIBUTE = None
     NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", ""]
```

Comment:
Yes, only for expression data. See also #1221

For clinical data this list is basically a subset of what is allowed in the java layer. So no, 'na' should not be in this list (since it is also not in https://github.com/cBioPortal/cbioportal/blob/rc/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java#L63)